### ISSUE TITLE
fails to load some modules with 'Not a HASH reference'

### ISSUE THREAD

**Issue by stuart-little** [niner/Inline-Perl5#158]
My script:

```
#!/usr/bin/env perl6
use v6;

use Math::BigFloat:from<Perl5>;
```
running it:
```
===SORRY!=== Error while compiling /root/bin/p6.p6
Not a HASH reference at /usr/share/perl/5.30/Math/BigFloat.pm line 1202.

at /root/bin/p6.p6:4
```
---
Version (`raku -v`)
```
Welcome to Rakudo(tm) v2020.11-99-gc4c4ba98c.
Implementing the Raku(tm) programming language v6.d.
Built on MoarVM version 2020.11-47-g1c7358004.
```

**Comment by niner** [comment]
Fixed in commit 6eb8dad906a310b92680b573697d9454215e0f90

Thanks for the report!

**Comment by stuart-little** [comment]
Just confirming that this works fine now if I install directly from this repo (rather from the current "official" `zef` distribution `0.50`). However, see [this issue](https://github.com/niner/Inline-Perl5/issues/159) on some problems installing from this repo vs directly via `zef install Inline::Perl5`.
